client/admin/main.js

```diff
@@ -15,6 +16,11 @@ Vue.use(VeeValidate, {
 Vue.use(VueHotkey);
 Vue.use(VueVisible);
 
+Vue.filter('formatDate', (value, dateFormat = 'MM/DD/YY HH:mm') => {
+  if (!value) return;
+  return format(value, dateFormat);
```

Comment:
Technically if you want to guard it from empty input you can still do it in one line:
```js
return value && format(value, dateFormat);
```